crawlers/base_crawler.py

```diff
@@ -50,8 +52,8 @@ def fetch_data(self):
             job_list = self.get_job_list(soup)
 
             for job in job_list:
-                if not re.search(f".*{self.keyword}.*", str(job), re.IGNORECASE):
-                    print('pass')
+                if self.need_include_keyword and not re.search(f".*{self.keyword}.*", str(job), re.IGNORECASE):
```

Comment:
I think it is not a good idea to set `need_include_keyword` and do not check the keyword is really in job category or job description.
I prefer overwrite `fetch_data` in **MeetJobs** or write a new function(maybe names `check_keyword`) and overwrite in **MeetJobs** to check keyword is in job detail page.

Reply:
yes, I'm agree with @setmao.

To solve this issue, maybe we can refactor base crawler for all platform. 
1. First to crawl search pages and get job list.
2. Second is to crawl job detail page to check keyword exists.
3. using asyncio to speed up crawler.